diff --git a/main.h b/main.h
index 3269533..6275b25 100644
--- a/main.h
+++ b/main.h
@@ -36,7 +36,7 @@
 #define RHEL_RELEASE_VERSION(a, b) a<<8 & b
 #endif
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 12, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 13, 0)
 /* Taken from kernel 6.18.1: include/linux/device/devres.h
  * Removed size_dup(...) to support kernels older than 5.15. */
 static inline void *devm_kmemdup_array(struct device *dev, const void *src,
